lib/situation_room/site_controller.ex

```diff
@@ -0,0 +1,79 @@
+defmodule SituationRoom.Site.Controller do
+  alias SituationRoom.Site
+  use Plug.Router
+
+  plug(:match)
+  # If we are going to include plug.Parsers, they go right here
+  plug(:dispatch)
+
+  # Get a site by name
+  get "/name/:name" do
+    get_site_by(conn, name: conn.params["name"])
+  end
+
+  # Get a site by endpoint/url
+  get "/endpoint/:endpoint" do
+    get_site_by(conn, endpoint: conn.params["endpoint"])
+  end
+
+  # Get a site by id
+  get "/id/:id" do
+    get_site_by(conn, id: conn.params["id"])
+  end
```

Comment:
We should use [RESTful](https://www.codecademy.com/article/what-is-rest) routes for our objects